src/main/java/com/synopsys/integration/detect/configuration/DetectProperties.java

```diff
@@ -265,6 +266,14 @@ private DetectProperties() {
             .setHelp("The depth at which Detect will search for files generated by Bitbake.")
             .setGroups(DetectGroup.BITBAKE, DetectGroup.SOURCE_SCAN);
 
+    public static final DetectProperty<NoneEnumListProperty<BitbakeDependencyType>> DETECT_BITBAKE_DEPENDENCY_TYPES_EXCLUDED =
+        new DetectProperty<>(new NoneEnumListProperty<>("detect.bitbake.excluded.dependency.types", NoneEnum.NONE, BitbakeDependencyType.class))
```

Comment:
I have been using the following naming pattern `detect.<detector>.dependency.types.excluded`
Example: `detect.conan.dependency.types.excluded`
Not sure if what I chose is better, but we should be consistent.

Reply:
I'm happy to change mine if you thing this reads better.

Reply:
I like yours; I changed mine.